webapp/App.js

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+/**
+ * Parent Component that is Rendered to the Page.
+ * @param {*} props arbituary input properties for our application to use
+ */
+export const App = (props) => {
+    return (
+        <h1>Welcome {props.name}</h1>
+    )
+}
```

Comment:
Please add a [newline at the end of the file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

Reply:
commit 927ef05

Reply:
Updated with commit 21cd082